### ISSUE TITLE
Why does the audio just randomly turn off while i was rendering this "swag route lets gooo" ashley wave trials video?

### ISSUE THREAD

**Issue by fgjhge** [ZiLko/xdBot#257]
i couldnt upload video here because "This video is too big. Try again with a file less than 10MB." and its also because the video was 55 megabytes, so i'll use streamable for now: https://streamable.com/gxkrj6
why does this happen?

**Comment by ZiLko** [comment]
its not random u have to let the whole level run for it to record the whole audio

**Comment by fgjhge** [comment]
so i have to complete the level at 100% to get the whole audio?

**Comment by ZiLko** [comment]
u have to let the macro run when it says "recording audio" with big text on the screen

**Comment by fgjhge** [comment]
yeah and thats what i did, for some reason it just finishes after like 8 seconds of rendering.
i let the macro run with the big text "Recording Audio" and then after like 8 seconds it just says "Recording Finished"
